### PR TITLE
Remove oc tag comments and add auto promotion message

### PR DESCRIPTION
**Pull request by scoheb**

### CODE REVIEW

File: JenkinsfileStageMerge [hunk 1]
```diff
@@ -1,34 +1,91 @@
...
 
-    step([$class: 'GhprbPullRequestMerge', allowMergeWithoutTriggerPhrase: false, deleteOnMerge: false, disallowOwnCode: false, failOnNonMerge: false, mergeComment: 'testing the merge', onlyAdminsMerge: false])
+/**
+ * Based on tagMap, add comment to GH with
+ * instructions to manual commands
+ *
+ * @param map of tags
+ * @return
+ */
+def sendPRCommentforTags(imageOperationsList) {
```
Comment (arilivigni) [inline JenkinsfileStageMerge]: Shouldn't this be added to the PipelineUtils or some new library instead of in the Jenkinsfile?

**Comment by scoheb** [comment]
@arilivigni moved methods to PipelineUtils

**Comment by scoheb** [comment]
[test]

**Comment by scoheb** [comment]
[merge]

**Pull request merged**